src/jira-connector.ts

```diff
@@ -22,31 +23,44 @@ export class JiraConnector {
     });
   }
 
-  async getTicketDetails(key: string): Promise<JIRADetails> {
-    const issue: JIRA.Issue = await this.getIssue(key);
-    const {
-      fields: { issuetype: type, project, summary },
-    } = issue;
+  async getTicketDetails(key: string, pr_status: string): Promise<void> {
+    try {
+      const url = `/issue/${key}?fields=project,summary,issuetype`;
+      const response = await this.client.get<JIRA.Issue>(url);
+      const issue: JIRA.Issue = response.data;
+      const {
+        fields: { issuetype: type, project, summary },
+      } = issue;
```

Comment:
can you please explain what is the reason of merging two methods into one?
what is the benefint?

Reply:
I had some trouble in the beginning to have a try catch block so moved them to a sinle method. Now I have reverted them back to two methods and have the try catch implemented.